bears/vcs/git/GitCommitBear.py

```diff
@@ -172,15 +158,17 @@ def check_imperative(self, paragraph):
                      " maxent_treebank_pos_tagger averaged_perceptron_tagger`")
             return
 
-    def check_body(self, body_line_length, force_body, body):
+    def check_body(self, body,
+                   body_line_length: int=72,
```

Comment:
I copied the earlier value, but I think too that 72 is the standard for the body length^^